.github/workflows/test-job.yaml

```diff
@@ -81,13 +81,13 @@ jobs:
         skip-pkg-cache: true
 
     - name: Run goimports
-      run: test -z "$(set -o pipefail && $(go env GOPATH)/bin/goimports -l apiserver/ ray-operator/ cli/ | tee goimports.out)" || { cat goimports.out && exit 1; }
+      run: test -z "$(set -o pipefail && $(go env GOPATH)/bin/goimports -l apiserver/ cli/ $(find ./ray-operator -name "*.go" | grep -v zz_generated.deepcopy.go) | tee goimports.out)" || { cat goimports.out && exit 1; }
 
     - name: Open this to see how to fix goimports if it fails
       run: |
-        echo "Run command 'goimports -w apiserver/ ray-operator/ cli/' to correct your code format."
+        echo "Run command 'goimports -w apiserver/ cli/ $(find ./ray-operator -name "*.go" | grep -v zz_generated.deepcopy.go)' to correct your code format."
```

Comment:
Could you add a comment explaining why you skip `zz_generated.deepcopy.go` and add this link https://github.com/kubernetes-sigs/controller-tools/issues/559?